### PR TITLE
[SERVER] Update schema with matches, top track, and user location

### PR DESCRIPTION
**Pull request by brianjaustin**

### CODE REVIEW

File: server/lib/melody_match/accounts/user.ex [hunk 1]
```diff
@@ -11,19 +11,23 @@ defmodule MelodyMatch.Accounts.User do
   use Ecto.Schema
   import Ecto.Changeset
 
+  alias MelodyMatch.Tracks.TopTrack
+
   schema "users" do
     field :email, :string
     field :name, :string
+    field :last_location, :string
```
Comment (pthom2503) [inline server/lib/melody_match/accounts/user.ex]: I guess location is a string of lat and long that just will require specific parsing?

**Review by pthom2503 — COMMENTED** [review]
My one big question is top track. May just be a me confusion thing. Are we storing just the one top track for each user in the DB and then polling their spotify connection for anything more? Sorry, the elixir naming convention of singulars and plurals has me a little turned around.

File: server/lib/melody_match/accounts/user.ex (hunk 1 above)
Comment (brianjaustin) [reply server/lib/melody_match/accounts/user.ex]: Hmm, I think it might make sense to split this out actually

**Comment by brianjaustin** [comment]
@pthom2503 we'll be storing only the number one top song, since that will be used in matchmaking. The top ten endpoint is kinda a weird add-on that doesn't play nicely with our data model. If we really want to I could probably work it in better, but I didn't know we were planning that when I was designing this

**Review by pthom2503 — APPROVED** [review]

**Pull request merged**